backend/internal/core/domain/gems/gem.go

```diff
@@ -16,13 +16,14 @@ const (
 )
 
 type Gem struct {
-	ID         int64
-	Message    string
-	Latitude   float64
-	Longitude  float64
-	CreatorId  int64
-	CreatedAt  time.Time
-	ReceiverId int64
-	ReceivedAt *time.Time
-	Color      GemColor
+	ID         int64      `db:"id" goqu:"skipinsert"`
+	Message    string     `db:"message"`
+	Latitude   float64    `db:"latitude"`
+	Longitude  float64    `db:"longitude"`
+	CreatorId  int64      `db:"creator_id"`
+	CreatedAt  time.Time  `db:"created_at" goqu:"skipinsert"`
+	ReceiverId int64      `db:"receiver_id"`
+	ReceivedAt *time.Time `db:"received_at"`
+	Color      GemColor   `db:"color"`
+	Attachment []byte     `db:"attachment"`
```

Comment:
Technically results in the domain models being coupled to the database, but oh well.

Reply:
hmm, i would like to think of it as the database using the domain model so the dependency is still inwards (logically)